module/VuFind/src/VuFind/Search/Blender/Params.php

```diff
@@ -119,7 +119,8 @@ public function initFromRequest($request)
             'type',
             'sort',
             'filter',
-            'hiddenFilter'
+            'hiddenFilter',
```

Comment:
Do we really need to support both hiddenFilter (singular) and hiddenFilters (plural), or do we just need to change one to the other? The only reference I can find in the code to the singular form is a configuration setting used in combined search, but that gets mapped to the plural form when added to the request.